wallet/src/de/schildbach/wallet/util/CrashReporter.java

```diff
@@ -177,26 +179,33 @@ public int compare(final PackageInfo lhs, final PackageInfo rhs) {
     }
 
     public static void appendApplicationInfo(
-        final Appendable report,
-        final PackageInfoProvider packageInfoProvider,
-        final Configuration configuration,
-        final Wallet wallet
+            final Appendable report,
+            final PackageInfoProvider packageInfoProvider,
+            final Configuration configuration,
+            final Wallet wallet,
+            final WalletApplication application
```

Comment:
We shouldn't provide the entire `WalletApplication` to this dialog - I made some changes recently so that `PackageInfoProvider` has everything needed here.

Reply:
These changes were made.